server/metrics.go

```diff
@@ -7,41 +7,13 @@ import (
 )
 
 var (
-	statusDuration = prometheus.NewHistogramFrom(stdprometheus.HistogramOpts{
+	requestDuration = prometheus.NewHistogramFrom(stdprometheus.HistogramOpts{
```

Comment:
Do we even look at these -- they are quite close to duplicating the API request metrics.